tests/baselines/reference/conditionalTypes2.errors.txt

```diff
@@ -52,15 +49,13 @@ tests/cases/conformance/types/conditional/conditionalTypes2.ts(75,12): error TS2
         ~
 !!! error TS2322: Type 'Covariant<A>' is not assignable to type 'Covariant<B>'.
 !!! error TS2322:   Type 'A' is not assignable to type 'B'.
-!!! error TS2322:     'A' is assignable to the constraint of type 'B', but 'B' could be instantiated with a different subtype of constraint '{}'.
     }
     
     function f2<A, B extends A>(a: Contravariant<A>, b: Contravariant<B>) {
         a = b;  // Error
         ~
 !!! error TS2322: Type 'Contravariant<B>' is not assignable to type 'Contravariant<A>'.
 !!! error TS2322:   Type 'A' is not assignable to type 'B'.
-!!! error TS2322:     'A' is assignable to the constraint of type 'B', but 'B' could be instantiated with a different subtype of constraint '{}'.

```

Comment:
The previous error message is wrong (`{}` isn't "interesting" constraint), but shouldn't the user still see an error message here?

Reply:
Presumably the user still sees `Type 'A' is not assignable to type 'B'.`, but I haven't added the elaboration from the linked proposal of:
```
'B' could be instantiated with an arbitrary type which could be unrelated to 'A'.
```